src/Events/UserMarkedThreadsAsRead.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace TeamTeaTime\Forum\Events;
+
+use Illuminate\Database\Eloquent\Collection;
+use TeamTeaTime\Forum\Models\Category;
+
+class UserMarkedThreadsAsRead
```

Comment:
Seems to be a lot of duplication between a few of these classes. Would it not make sense to have a base class, which has user and the threads collection - then simply inherit/extend as needed.

- UserMarkedThreadAsRead
- UserViewingRecent
- UserViewingUnread